scripts/jira/cloner.py

```diff
@@ -203,30 +220,43 @@ def scan_issue(issue, connection):
         if has_fix_versions_label(issue):
             actions.append(Action(issue.key, "Remove Fix versions label", remove_needs_fix_version_label, issue=issue))
 
+    if get_sprint(issue) is None:
+        actions.append(Action(issue.key, "Issue needs to be included in a sprint", None))
+
     if is_original_issue(issue):
         actions.extend(scan_original_issue(issue, connection))
     else:
         actions.extend(scan_cloned_issue(issue, connection))
     return actions
 
+def query_build(issue, user):
+    query = JQL_FILTER_QUERY
+    if len(issue) > 0:
```

Comment:
issue is a string? So this could just be `if issue`?

Reply:
You are right, done.